lib/foods.js

```diff
@@ -1,15 +1,30 @@
+var foods;
```

Comment:
if we don't need this variable to be available within each function below, we may not want to set this here. the alternative would be to just pass the food collection around as arguments to your functions 👍